lib/platform/gitlab/index.ts

```diff
@@ -511,25 +531,43 @@ export async function ensureIssueClosing(title: string) {
   }
 }
 
-export async function addAssignees(iid: number, assignees: string[]) {
+export async function addAssignees(
+  iid: number,
+  assignees: string[]
+): Promise<any> {
   logger.debug(`Adding assignees ${assignees} to #${iid}`);
   try {
-    const assigneeId = (await api.get(`users?username=${assignees[0]}`)).body[0]
+    let assigneeId = (await api.get(`users?username=${assignees[0]}`)).body[0]
       .id;
-    let url = `projects/${config.repository}/merge_requests/${iid}`;
-    url += `?assignee_id=${assigneeId}`;
+    let url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
     await api.put(url);
```

Comment:
So first step in all cases is to use the "single assignee" endpoint to assign the first assignee?

Reply:
That's the idea to remain safe on having at least 1 in case of failure.